### PR TITLE
E2E Starter branch

### PR DESCRIPTION
**Pull request by android-dev-lxl**

### CODE REVIEW

**Review by johnshea — COMMENTED** [review]
MainActivity.kt - Is this call still needed `WindowCompat.setDecorFitsSystemWindows(window, false)`? It is part of the e2e code in `Theme.kt` and appears to be redundant.

File: app/src/main/java/com/example/marsphotos/ui/screens/HomeScreen.kt [hunk 1]
```diff
@@ -16,18 +16,22 @@
...
+import androidx.compose.ui.unit.dp
 import com.example.marsphotos.R
 import com.example.marsphotos.ui.theme.MarsPhotosTheme
 
 @Composable
 fun HomeScreen(
-    marsUiState: String, modifier: Modifier = Modifier
+    marsUiState: String,
+    modifier: Modifier = Modifier,
+    contentPadding: PaddingValues = PaddingValues(0.dp),
```
[the remark this answers is not in the document]
Comment (android-dev-lxl) [reply app/src/main/java/com/example/marsphotos/ui/screens/HomeScreen.kt:34]: https://github.com/google-developer-training/basic-android-kotlin-compose-training-mars-photos/pull/83#discussion_r1310960047

**Review by johnshea — CHANGES_REQUESTED** [review]
There is an issue with how the UI for e2e is done. In starter_e2e, the surface is running under the topappbar which is not happening in branch starter.

Note - I added additional Text composables to check this.
Screenshot 2023-09-19 at 11 59 12 AM

Screenshot 2023-09-19 at 11 56 37 AM
Screenshot 2023-09-19 at 12 05 01 PM

**Review by johnshea — APPROVED** [review]
lgtm

**Review by johnshea — APPROVED** [review]
lgtm

**Pull request merged**